### ISSUE TITLE
nbrobj and nbrobjcum columns of taxonomy table have a non-intuitive content

### ISSUE THREAD

**Issue by grololo06** [ecotaxa/ecotaxa_dev#648]
While it might seem that `nbrobj` simply contains the number of objects with the category, in fact it's (see cron.py source) the number of *validated* objects in *public* projects.
If the count has to remain as such, the column needs to be renamed and API doc aligned.
Otherwise, the code filling this value must be corrected.

**Comment by grololo06** [comment]
Looks like the SQL is wrong:
```
ecotaxa=# select sum(nbrobj) from taxonomy;
   sum    
----------
 46550852
(1 ligne)

ecotaxa=# select nbrobj,nbrobjcum from taxonomy where parent_id is null;
 nbrobj | nbrobjcum 
--------+-----------
  31433 |  16417414
  11412 |     15978
  10848 |  23271490
(3 lignes)

```

**Comment by grololo06** [comment]
With last code, not deployed so far:
`
ecotaxa4=# select sum(nbrobj) from taxonomy;
   sum    
----------
 68121696
(1 ligne)

Temps : 417,550 ms
ecotaxa4=# select nbrobj,nbrobjcum from taxonomy where parent_id is null;
 nbrobj | nbrobjcum 
--------+-----------
  33475 |  28426271
  11716 |   2591617
  18243 |  37103808
(3 lignes)

Temps : 0,525 ms
ecotaxa4=# select 28426271+2591617+37103808;
 ?column? 
----------
 68121696
(1 ligne)

Temps : 0,485 ms
`

**Comment by grololo06** [comment]
Fixed in 2.5.11 (same column names but more accurate values)
